python/plugins/processing/core/ProcessingLog.py

```diff
@@ -31,6 +31,7 @@
 import datetime
 from processing.tools.system import userFolder
 from processing.core.ProcessingConfig import ProcessingConfig
+from processing.core import Processing
```

Comment:
This introduces circular dependency. ProcessingLog imports Processing and in same time Processing imports ProcessingLog. IMHO it is better to create method in the ProcessingLog class for i18n support, like in we already have in ProcessingConfig
```
    def tr(self, string, context=''):
        if context == '':
            context = 'ProcessingLog'
        return QCoreApplication.translate(context, string)
```